### ISSUE TITLE
[Question] Github actions - no tests found

### ISSUE THREAD

**Issue by michelle-choi1** [microsoft/playwright#19290]
Hi, when I run my tests on a cron through Github actions, I'm getting automatic failures on my tests saying "no tests found" even though they were previously working fine on the cron before I reorganized the files, and when I now run them on the terminal on my computer, they run fine. I have different sets of tests in each folder, and I have each folder running on different jobs. 

error:
```
=================
 no tests found.
=================
```

According to https://github.com/microsoft/playwright/issues/11060#issuecomment-999933284 I changed the config file to look like this:
```js
const config = {
  testDir: __dirname,
  testMatch: '**/*.spec.js',
  testIgnore: '**/wip-tests/**/*',
```
and here is my folder structure:

- playwright-tests
   - live-tests
      - long-term
         - setup
            - store1-setup.spec.js
            - store2-setup.spec.js
         - tests
            - store1-tests.spec.js
            - store2-tests.spec.js
      - stream-a
         - tests.spec.js
         - ...
      - stream-b
         - tests.spec.js
         - ...
      - stream-c
         - tests.spec.js
         - ...
   - wip-tests
   - playwright.config.js

thanks in advance for ur help!!

**Comment by michelle-choi1** [comment]
thanks @aslushnikov yeah, thank u so much for helping me look into this! uploaded it here - https://github.com/michelle-choi1/effective-octo-garbanzo

**Comment by michelle-choi1** [comment]
@aslushnikov I also just ran the small repo on my local and it seemed to work. do you think it might be more of an issue with github actions vs tinkering with my playwright project setup?

**Comment by michelle-choi1** [comment]
@aslushnikov thanks for checking! it was a matter of spacing so I fixed it, and it still didn't work so I've resorted for now to commenting out all the other jobs and checking if having one job will cause the tests to be recognized. The tests were detected when there was only one to two jobs working in the cron, so I'll have to investigate further to see why it's doing that. But as for now I don't think it's an issue with my playwright config and more of an issue with my cron file so I'll close this for now. Thank you for your time!
